spec/features/authentication_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.feature 'Authentication', type: :feature do
+  let(:admin) { build(:user) }
```

Comment:
```ruby
  let!(:admin) { create(:user) }
  let!(:user) { create(:user) }
  let!(:menu) { create(:menu, :with_meals, meal_count: 1) }
```

Skip trash above to `Then you write your tests:...`
[source](http://bodrovis.tech/en/categories/ruby-rails/posts/rails-and-rspec-difference-between-let-and-let)

Reply:
Does he need `let!` instead of `let` here?

Reply:
@kushniryb 
hm, yep, I think you're right, 
but anyway `before` hook is useless here